minor doc fixes (#59)
canonical/grafana-k8s-operator: README.md

observability toolkit.

Grafana allows you to query, visualize, alert on, and visualize metrics from mixed datasources in configurable

> dashboards for observability. This repository contains a [Juju](https://jaas.ai/) Charm for deploying the visualization component of Grafana in a Kubernetes cluster.

Reviewer: The link should go to juju.is, not jaas.ai

Author: Thanks! Fixed in next commit.

The default password is randomized at first install, and can be retrieved with:
```bash
juju run-action grafana-k8s/0 get-admin-password --wait
```

View the dashboard in a browser:
1. `juju status` to check the IP of the running Grafana application
2. Navigate to `http://IP_ADDRESS:3000`

> 3. Log in with the credentials username=admin, password=<from run-action> (these credentials are configurable at deploy time)

Reviewer: I don’t think this is understandable. Please add a section about retrieving the admin pwd if we do not have one, and ljnk it from here

Author: The action is described in line 37.

2. Navigate to `http://IP_ADDRESS:3000`

> 3. Log in with the credentials username=admin, password=<from run-action> (these credentials are configurable at deploy time)

Reviewer 2: ```suggestion
3. Log in with the username `admin`, and the password you got from the `get-admin-password` action.
```
